src/App.jsx

```diff
@@ -1,35 +1,66 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import * as React from "react";
+import TodoList from "./TodoList.jsx";
+import AddTodoForm from "./AddTodoForm.jsx";
+import { useState, useEffect } from "react";
+
+// const title = 'React';
+/* Remembe to change the code inside this function to varaibles */
+const useSemiPersistentState = () => {
+  var onLoadList = JSON.parse(localStorage.getItem("savedTodoList"));
```

Comment:
I recommend avoiding the use of `var` and instead stick to either `let` or `const`.

https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994